cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -201,7 +201,11 @@ export class MigrationWave {
         const currentStartDate =
             this.startDate.getTime() !== date.getTime() ? this.startDate : new Date();
         const currentMonth = currentStartDate.toLocaleString("en-us", { month: "long" });
-        cy.get(MigrationWaveView.startDateInput).next("button").click();
+        cy.get(MigrationWaveView.startDateInput)
+            .parents(MigrationWaveView.generalDatePicker)
```

Comment:
Consider to use `closest()` instead of `parents()`, it can be more flexible in case of next structure change

Reply:
done